src/Symfony/Bundle/FrameworkBundle/Resources/config/assets.php

```diff
@@ -77,13 +77,15 @@
             ->abstract()
             ->args([
                 abstract_arg('manifest path'),
+                false,
             ])
 
         ->set('assets.remote_json_manifest_version_strategy', RemoteJsonManifestVersionStrategy::class)
             ->abstract()
             ->args([
                 abstract_arg('manifest url'),
                 service('http_client'),
+                false,
```

Comment:
I think these should both be abstract args - as we fill both in always

Reply:
I used a fixed value to prevent breaking compatibility in the eventuality someone extends this abstract définition.

Reply:
Is is something I must change ?